### PR TITLE
add Cactusphere Basic Version device model

### PR DESCRIPTION
**Pull request by naomitodori**
### Company Info

Atmark Techno, Inc.
https://www.atmark-techno.com/

### IoT Plug and Play Device Info

Cactusphere
https://armadillo.atmark-techno.com/cactusphere

### Model Submission Goals

Device certification

### CODE REVIEW

**Comment by rido-min** [comment]
@Shin-yaKoga  @ReikaYamazaki as owners of the atmark-techno namespace can you review/approve this PR?

**Comment by Shin-yaKoga** [comment]
Dear Ricardo, Yes, Naomi TODORI is our colleague and she can review/approve it.

File: dtmi/atmark_techno/cactusphere/cactusphere_basic-1.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      "description": "RTApp version of the device.",
+      "writable": false
+    },
+    {
+      "@type": ["Telemetry", "Event"],
+      "name": "SphereWarning",
+      "schema": "string",
+      "displayName": "Error information",
+      "description": "Error information of the device.",
+      "writable": false
```
Comment (rido-min) [inline dtmi/atmark_techno/cactusphere/cactusphere_basic-1.json:78]: `writable` is a Property annotation, it's not for Telemetry

**Review by rido-min — CHANGES_REQUESTED** [review]
remove writable from telemetry

File: dtmi/atmark_techno/cactusphere/cactusphere_basic-1.json (hunk 1 above)
Comment (naomitodori) [reply dtmi/atmark_techno/cactusphere/cactusphere_basic-1.json]: It has been deleted. Please confirm it is.

**Review by rido-min — APPROVED** [review]

**Pull request merged**